File: PRAK204-2510817210005-Daffa-Dziibaan-Khoiri.c
#include <stdio.h>

int main()
{
    float jari, tinggi, volume, luas, keliling, phi = 3.142857;
    scanf ("%f %f", &jari, &tinggi);
    volume = phi * (jari * jari) * tinggi;
    luas = 2 * phi * jari * (jari + tinggi);
    keliling = 2 * phi * jari;
    printf ("Volume = %.2f\n", volume);
    printf ("Luas = %.2f\n", luas);
    printf ("Keliling = %.2f\n", keliling);

    return 0;
}

File: PRAK204-2510817210005-Daffa-Dziibaan-Khoiri.py

import sys
input_data = sys.stdin.read().strip()
values = input_data.split()
jari, tinggi = map(float, values[:2])
phi = 3.142857
volume = phi * jari * jari * tinggi
luas = 2 * phi * jari * (jari + tinggi)
keliling = 2 * phi * jari

print(f"Volume : {volume:.2f}")
print(f"Luas    : {luas:.2f}")
print(f"Keliling: {keliling:.2f}")